### ISSUE TITLE
error: comparison between pointer and integer

### ISSUE THREAD

**Issue by m6w6** [awesomized/libmemcached#34]
* *Reported as:* [lp:1284687](https://bugs.launchpad.net/bugs/1284687)
 * *Reported by:* [lp:~dev.zone](https://launchpad.net/~dev.zone)
 * *Reported at:* 2014-02-25T15:34:07Z

Imported from Launchpad using lp2gh.

------------------------------------

  CXX      clients/memflush.o
clients/memflush.cc:42:19: error: comparison between pointer and integer ('char *' and 'int')
  if (opt_servers == false)
      ~~~~~~~~~~~ ^  ~~~~~
clients/memflush.cc:51:21: error: comparison between pointer and integer ('char *' and 'int')
    if (opt_servers == false)
        ~~~~~~~~~~~ ^  ~~~~~
2 errors generated.
make[1]: *** [clients/memflush.o] Error 1
make: *** [install] Error 2

**Comment by m6w6** [comment]
* *Comment by:* [lp:~mauriciow95](https://launchpad.net/~mauriciow95):
 * *Created at:* 2014-05-13T21:17:02Z

-----
I'm having the same issue.
libmemcached-1.0.18
Xcode 5.1.1
OS X 10.9.2

**Comment by m6w6** [comment]
* *Comment by:* [lp:~sonthehottest](https://launchpad.net/~sonthehottest):
 * *Created at:* 2014-10-05T09:20:23Z

-----
I do have the same issue.
libmemcached-1.0.18
Xcode 6.0.1
OSX 10.9.5

Any help would be appreciated

**Comment by m6w6** [comment]
* *Comment by:* [lp:~sonthehottest](https://launchpad.net/~sonthehottest):
 * *Created at:* 2014-10-05T09:29:56Z

-----
changing the 'false' to 'NULL' in memflush.cc file resulted in success for 'make' command on my computer.

hope it helps.

**Comment by m6w6** [comment]
Fixed in dc1e22ba362663bf052e7cfbbdf933bb462a7006
